### PR TITLE
DROOLS-4545 : GDST V&V: Clean up situations where the webworker fails

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-4545

https://github.com/kiegroup/drools/pull/2546
https://github.com/kiegroup/kie-wb-common/pull/2909
https://github.com/kiegroup/drools-wb/pull/1233

### CODE REVIEW

**Review by jomarko — APPROVED** [review]

**Review by yesamer — APPROVED** [review]
@Rikkola fine for me, the only suggestion I have is to add a test for `IllegalVerifierStateIssue` constructor if possible.

**Comment by Rikkola** [comment]
Thank you. Test added.

**Comment by Rikkola** [comment]
Jenkins execute downstream build

**Comment by Rikkola** [comment]
Problem was:

> [ERROR] Failed to execute goal on project jira-workitem: Could not resolve dependencies for project org.jbpm.contrib:jira-workitem:jar:7.28.0-SNAPSHOT: Failure to find com.atlassian.jira:jira-rest-java-client:jar:1.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

**Comment by Rikkola** [comment]
> [ERROR] Failed to execute goal on project jira-workitem: Could not resolve dependencies for project org.jbpm.contrib:jira-workitem:jar:7.28.0-SNAPSHOT: Failure to find com.atlassian.jira:jira-rest-java-client:jar:1.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project jira-workitem: Could not resolve dependencies for project org.jbpm.contrib:jira-workitem:jar:7.28.0-SNAPSHOT: Failure to find com.atlassian.jira:jira-rest-java-client:jar:1.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced

Jenkins compile downstream build

**Comment by Rikkola** [comment]
Jenkins compile downstream build

**Comment by Rikkola** [comment]
Jenkins execute downstream build

File: drools-verifier/drools-verifier-api/src/main/java/org/drools/verifier/api/reporting/IllegalVerifierStateIssue.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.util.HashSet;
+
+public class IllegalVerifierStateIssue
+        extends Issue {
+
+    public IllegalVerifierStateIssue() {
+        super(Severity.ERROR,
+              CheckType.ILLEGAL_VERIFIER_STATE,
+              new HashSet<Integer>()
```
Comment (jomarko) [inline drools-verifier/drools-verifier-api/src/main/java/org/drools/verifier/api/reporting/IllegalVerifierStateIssue.java]: @Rikkola investigating coverage I realized we could use `Collections.emptySet` or `Collections.EMPTY_SET`, what do you think?

Comment (Rikkola) [reply drools-verifier/drools-verifier-api/src/main/java/org/drools/verifier/api/reporting/IllegalVerifierStateIssue.java]: Yes makes sense. The list should not be modifiable after the issue is created.

**Pull request merged**